### ISSUE TITLE
status of node-ari-client

### ISSUE THREAD

**Issue by jmls** [asterisk/node-ari-client#132]
honestly not trying to be a troll - but I need to know Is this project still active ? 

Last commit nearly 8 months ago, unresolved issues and last mention of an asterisk version is 15.6 . 

I'm considering a new project with asterisk and node and need to find the most appropriate library and don't want to get stuck with an non-maintained one

thanks

**Comment by digitaltoast** [comment]
I was wondering the same - a few obsolete package warnings on install:

```
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated swagger-client@2.0.26: No longer maintained, please upgrade to swagger-client@3.
npm WARN deprecated sprintf@0.1.1: The sprintf package is deprecated in favor of sprintf-js.
```

**Comment by cervajs** [comment]
+1

**Comment by carlos-menezes** [comment]
+1. Any plan for types? I'm willing to fork this project and work on it.

**Comment by dioris-moreno** [comment]
Today was published @types/ari-client to NPM (my contribution). I also published this project https://github.com/dioris-moreno/ari-typescript-examples that includes the examples of node-ari-client library, converted to TypeScript, and using these new type definitions. I hope this will help to facilitate the development of applications based on ARI.

**Comment by raarts** [comment]
There's more activity now, see https://github.com/asterisk/node-ari-client/pull/137.
@gtjoseph merged a pull request and stated they lost track of this repository.

**Comment by ghost** [comment]
UPDATE
-------

 OK, after a couple of hours reading mailing list and more stuff, I now understand that the version of Swagger API used **internally** by Asterisk (the server) is the part that is obsolete, so the npm **client** package cannot use the current API version.
So it looks like everything *will* actually work, albeit with a log full of warnings.

Original message
-------

I was just wondering if there were still plans to update this to the current version of Swagger as we roll into 2021?

Thanks!

> I was wondering the same - a few obsolete package warnings on install:
> 
> ```
> npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
> npm WARN deprecated swagger-client@2.0.26: No longer maintained, please upgrade to swagger-client@3.
> npm WARN deprecated sprintf@0.1.1: The sprintf package is deprecated in favor of sprintf-js.
> ```

**Comment by raarts** [comment]
> it looks like everything will actually work, albeit with a log full of warnings.

Yes. I use @dioris-moreno 's typescript version, and haven't found any significant problems yet.

**Comment by ghost** [comment]
Hi @raarts - Many thanks for the heads-up on that version. REALLY helpful. Thanks!
